app/views/index/normal.phtml

```diff
@@ -110,14 +110,13 @@ $today = @strtotime('today');
 	if ($nbEntries > 0):
 		call_user_func($this->callbackBeforePagination, $this, $nbEntries, $lastEntry);
 		$this->renderHelper('pagination');
-?></div><?php
+?></main><?php
 	else:
 		ob_end_clean();	//Discard the articles headers, as we have no articles
 ?>
-<div id="stream" class="prompt alert alert-warn normal">
+<main id="stream" class="prompt alert alert-warn normal">
 	<h2><?= _t('index.feed.empty') ?></h2>
 	<a href="<?= _url('subscription', 'add') ?>"><?= _t('index.feed.add') ?></a><br /><br />
-</div>
+</main>
 <?php endif; ?>
-
```

Comment:
Please keep it.

Reply:
fixed,  but I do not understand why it is necessary